cmd/notary/config.json

```diff
@@ -0,0 +1,7 @@
+{
+	"remote_server": {
+		"url": "https://localhost:4443",
```

Comment:
That's true - it doesn't for users of `docker-machine`, though, unless they SSH forward or remap localhost to the VM.  I'm fine either way, so long as we document a single way to get things to work on both docker-on-localhost and docker-machine.

If we want to go with localhost, then I'd suggest SSH-forwarding for those who use docker-machine (since remapping localhost seems kind of ugly):  `ssh docker@$(docker-machine ip default) -L 4443:localhost:4443` ?